content/topics/error-handling/_index.md

```diff
@@ -0,0 +1,103 @@
+---
+content_type: topic
+title: Error handling best practices
+tags:
+  - javascript
+flavours:
+  - javascript
+---
+
+## Errors are useful
+
+There's a reason why errors are in almost all if not all languages. They serve to indicate that something bad and unintended happens. They don't exist to just be squashed or just logged.
+
+Errors that occur usually stop and terminate the program. The most common way of handling errors in javascript is by using the [try catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) statement. This is used to try and gracefully handle the error or use info from the error to handle it accordingly.
+
+```js
+try {
+  // critical code that may fail goes here
+} catch (error) {
+  // - if an error occurs in the try the error is caught and made available here
+  // - this is where you can call verify certain errors, notify the user .etc.
+}
+```
+
+## Common bad ways people/tutorials handle errors
```

Comment:
i will suggest this heading be more concise like the others, **bad error handling practices** or **Bad error handling** these are  brief and comprehensive.